sdk/src/builder.rs

```diff
@@ -1422,6 +1418,77 @@ impl Builder {
         Ok(self)
     }
 
+    #[async_generic(async_signature(
+        &mut self,
+        time_authority_url: &str,
+        store: &mut Store,
+        http_resolver: &impl AsyncHttpResolver,
+    ))]
+    fn maybe_add_timestamp(
```

Comment:
Since this function only adds the timestamp for parent ingredient we should rename this function.  Where is the capability to add timestamps for any manifest in the store?